test/e2e/service_bootstrap.py

```diff
@@ -116,8 +117,19 @@ def service_bootstrap() -> Resources:
             },
         ),
         ESMQueue=Queue(
-            "ack-lambda-controller-queue"
+            "ack-lambda-controller-queue",
+        ),
+        EICRole=Role(
+            "ack-lambda-controller-eic-role",
+            principal_service="lambda.amazonaws.com",
+            managed_policies=EIC_ROLE_POLICIES,
         ),
+        EICQueueOnSuccess=Queue(
+            "ack-lambda-controller-queue" 
```

Comment:
Should we change this to `ack-lambda-controller-function-queue-eic-onsuccess`? and apply same logic for the on failure?

Reply:
Done